pkg/reconciler/inmemorychannel/controller/inmemorychannel_test.go

```diff
@@ -83,6 +83,8 @@ var (
 		URL:  apis.HTTP("test-imc-kn-channel.test-namespace.svc.cluster.local"),
 	}
 
+	channelAudience = fmt.Sprintf("messaging.knative.dev/inmemorychannel/%s/%s", testNS, imcName)
```

Comment:
Technically, the unique name is not the kind but resource name which is the plural form
https://github.com/knative/eventing/blob/0af3ef6cbb706acdf354aeb8c5c347e81b6c4d85/config/channels/in-memory-channel/resources/in-memory-channel.yaml#L17